### ISSUE TITLE
Not all keys can be pressed together

### ISSUE THREAD

**Issue by eclab** [flit/MidiKeys#3]
It's great to see MidiKeys revived.  I'm highly reliant on it in testing my own open source software.  I'll post known bugs in 1.8 which are still present in 1.9, and maybe some new ones.

First up: if you hold q and t down (and no doubt other combos), you cannot press [ or -

High Sierra, Mid 2015 Macbook Pro.

**Comment by flit** [comment]
Thanks for adding all the bug reports!

Some of the key combos that don't work might be due to hardware limitations. Have you tried with a USB keyboard?

**Comment by eclab** [comment]
I have not tried with an external keyboard and due to the Covid lockdown I won't be able to any time soon. It is possible that it's some kind of multiplexing thing: but on a modern Macbook laptop this would seem... surprising?

**Comment by flit** [comment]
It certainly used to be an issue with some keyboards. Just worth double checking whenever you get a chance. Of course, I'll try to reproduce it, as well.

**Comment by eclab** [comment]
Testing on the MacOS Keyboard Viewer suggests that it is indeed a multiplexing issue, likely hardware.  Other keys which wont' work with q+t include 1, a, z, 5, g, and b.  Basically anything on the q and t column.  I think you can close this one out.

**Comment by flit** [comment]
Good idea using the keyboard viewer, thanks for testing more. I think I'll keep the issue open and pin it so other users see it.
